Entities/MainObjects/Player.tscn

```diff
@@ -72,7 +72,6 @@ resource_local_to_scene = true
 size = Vector2(18, 38)
 
 [sub_resource type="CanvasItemMaterial" id="2"]
-blend_mode = 5
```

Comment:
explain/revert

Reply:
I can't access this value in the editor. Weird, but I'll revert this.